entrypoint.sh

```
@@ -1,43 +1,30 @@
-
-# Check if migrations need to be applied
-if check_migrations; then
-    echo "Migrations need to be applied. Running initialization tasks."
-
-    # Run migrations
-    echo "Migration script is running"
-    python manage.py migrate
+# Load .env if it exists
+if [ -f /blt/.env ]; then
+  echo "Loading environment variables from .env..."
+  export $(grep -v '^#' /blt/.env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)